blablahelp-frontend/src/views/user/OfferForm.tsx

```diff
@@ -357,18 +360,21 @@ export default function OfferForm() {
                             timeTo={timeTo}
                         />
 
-                        <Typography component={'span'} sx={{fontSize: "1.2rem"}}>Maximale Anzahl an
-                            Mitshoppern {maxMitshoppers}</Typography>
-                        <ButtonGroup variant="outlined"
-                                     onClick={(e) => {
-                                         const val = (e.target as HTMLInputElement).value;
-                                         setMaxMitshoppers(Number(val))
-                                     }}
-                                     aria-label="Maximale Anzahl an Mitshoppern">
-                            <Button value={"1"}>Eins</Button>
-                            <Button value={"2"}>Zwei</Button>
-                            <Button value={"3"}>Drei</Button>
-                        </ButtonGroup>
+                        <FormControl>
+                            <FormLabel id=",maxMitshopper-radio-buttons-group-label">
+                                Maximale Anzahl an Mitshoppern</FormLabel>
+                            <RadioGroup aria-labelledby="maxMitshopper-radio-buttons-group-label" row
+                                        value={maxMitshoppers ?? ""}
+                                        onChange={(e) => {
+                                            const val = (e.target as HTMLInputElement).value;
+                                            setMaxMitshoppers(Number(val))
+                                        }}
+                            >
+                                <FormControlLabel value={"1"} control={<Radio/>} label="Eins"/>
+                                <FormControlLabel value={"2"} control={<Radio/>} label="Zwei"/>
+                                <FormControlLabel value={"3"} control={<Radio/>} label="Drei"/>
+                            </RadioGroup>
+                        </FormControl>
 
```

Comment:
You could really (following the "react-way") outsource some components...would make the code more readable... I would love to give you advice, but I will not lay my hands on a monster of this size.. 😅